nltk/collocations.py

```diff
@@ -122,6 +123,22 @@ def above_score(self, score_fn, min_score):
                 break
 
 
+
+def _build_new_documents(documents, window_size, pad_left=False, pad_right=False, pad_symbol=None):
```

Comment:
@dimazest, @jnothman – there's other places where `None` in a sequence of tokens is treated specially, e.g. null alignments. Also, this is the status quo. I think a change like this would need to be made after a careful review of the rest of the codebase and a discussion on nltk-dev.